content/en/docs/installation/guide/install-on-gke-operator.md

```diff
@@ -253,90 +158,52 @@ sudo mv kustomize /usr/local/bin/
-
-Here is an example `profiles-patch.yml` with Kustomize turned on and basic form authentication configured.
-
-**profiles-patch.yml**
-
-```yaml
-apiVersion: spinnaker.armory.io/v1alpha2
-kind: SpinnakerService
-metadata:
-  name: spinnaker  # name doesn't matter since this is a patch
-spec:
-  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
-  spinnakerConfig:
-    # spec.spinnakerConfig.profiles - This section contains the YAML of each service's profile
-    profiles:
-      clouddriver: {} # is the contents of ~/.hal/default/profiles/clouddriver.yml
-      # deck has a special key "settings-local.js" for the contents of settings-local.js
-      deck:
-        # settings-local.js - contents of ~/.hal/default/profiles/settings-local.js
-        # Use the | YAML symbol to indicate a block-style multiline string
-        settings-local.js: |
-          window.spinnakerSettings.feature.kustomizeEnabled = true;
-          window.spinnakerSettings.feature.artifactsRewrite = true;
-          window.spinnakerSettings.authEnabled = true;
-      echo: {}    # is the contents of ~/.hal/default/profiles/echo.yml
-      fiat: {}    # is the contents of ~/.hal/default/profiles/fiat.yml
-      front50: {} # is the contents of ~/.hal/default/profiles/front50.yml
-      gate:
-        security:
-          basicform:
-            enabled: true
-          user:
-            name: spin
-            password: spin4u99
-      igor: {}    # is the contents of ~/.hal/default/profiles/igor.yml
-      kayenta: {} # is the contents of ~/.hal/default/profiles/kayenta.yml
-      orca: {}    # is the contents of ~/.hal/default/profiles/orca.yml
-      rosco: {}   # is the contents of ~/.hal/default/profiles/rosco.yml
-```
-
-Alternately, you can include a separate patch file for each component and then reference the component in the `kustomization.yml` file. See the Kustomize [docs](https://github.com/kubernetes-sigs/kustomize/tree/master/docs) for details.
+The `spinnaker-kustomize-patches` repository contains several examples for
+adding authentication to your cluster. Consult the examples in the `security`
+directory and choose the most appropriate example for your environment. For
+example, to enable basic auth, modify the `security/patch-basic-auth.yml` by
+changing the username to one of your choosing. Then, add the file path to your
+`kustomization.yml` file in the `patchesStrategicMerge` section. Finally,
+modify the `secrets-example.env` file to choose a password unique to you, and
+run the `./create-secrets.sh` script to create Kubernetes credentials in your
+cluster.
```

Comment:
Received feedback from a customer via Ryan Cartwright "The docs missed that you need to ensure the right Auth Scopes are enabled on the GKE node pools, otherwise you hit Authentication issues trying to write to Cloud Storage for logging"  -- not sure how to include this?